qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/completions/QuteCompletionsForExpression.java

```diff
@@ -251,17 +252,18 @@ private void fillCompletionFields(ResolvedJavaTypeInfo resolvedType, Range range
 			// class A extends B {String foo;}
 			// class B {String foo;}
 			if (!existingProperties.contains(fieldName)) {
+				String simpleFieldName = getSimpleType(fieldName);

```

Comment:
Why simpleFieldName? Its the type we must simplified not the name.

Reply:
You can see below that this fix does the following for `CASE_INSENSITIVE`:
![Screenshot from 2021-12-16 14-27-37](https://user-images.githubusercontent.com/26389510/146436035-bca044c3-3367-425e-90bd-382527d6841a.png)
Which is now of type `Comparator<String>` rather then `java.util.Comparator<java.lang.String>` as it was before.

Reply:
It would be ideal if my feature is tested before review, since I covered the case we mentioned in the original issue.

Reply:
I think the fix should be just:

`item.setLabel(fieldName + " : " + getSimpleType(field.getType()));`

fieldName doesn't contains some '.', so you don't need to simplify field name.

> It would be ideal if my feature is tested before review, since I covered the case we mentioned in the original issue.

Indeed you cover it, but you don't need to simply field name.